examples/calculator/Makefile

```diff
@@ -1,6 +1,8 @@
 all:
 	mkdir -p bin/
-	../../bin/nitc --dir bin/ src/calculator_gtk.nit src/calculator_test.nit
+
+	# Compile in global mode to silence warnings on unused GTK deprecated
+	../../bin/nitc --global --dir bin/ src/calculator_gtk.nit src/calculator_test.nit
```

Comment:
`--semi-global` still cause deprecated warnings...

and I prefer `--global` to `nitg` because it is explicit and...  reading the script `nitg`, it doesn't make it global, does it?